### PR TITLE
CUDA BFloat16 Dropout

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: test/test_nn.py [hunk 1]
```diff
@@ -10055,7 +10055,7 @@ def test_Dropout(self, device):
         input = torch.Tensor(1000)
```
Comment (ngimel) [inline test/test_nn.py:10055]: how the hell does this work? It's an uninitialized tensor!
Also, looks like dropout is not tested for datatypes other than default (double?float?) and bfloat16 ? Seems dangerous.

Comment (zasdfgbnm) [reply test/test_nn.py]: fixed

**Comment by zasdfgbnm** [comment]
ping @ngimel

**Comment by ngimel** [comment]
Sorry, this needs a rebase after dropout fix.

**Comment by zasdfgbnm** [comment]
@ngimel Rebased

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**